lib/signalfx/signalflow/client.rb

```diff
@@ -0,0 +1,58 @@
+require 'thread'
+
+require_relative "./websocket"
+
+# A SignalFlow client that supports swappable .
+class SignalFlowClient
+  def initialize(api_token, api_endpoint, stream_endpoint)
```

Comment:
Why do you need the api_endpoint? You don't seem to be using it.

Reply:
It is in anticipation of the implementation of the HTTP-only SignalFlow connection (which I'm not going to do right now).